archiwarep5.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Archiware P5",
+    "plugin_schema": "2",
+    "release": "11.2-RELEASE",
+    "artifact": "https://github.com/IgorC-test/P5-freenas.git",
+    "pkgs": [
+"irc/quassel-core"
+    ],
+    "packagesite": "http://pkg.cdn.trueos.org/iocage",
+    "fingerprints": {
+        "iocage-plugins": [
+            {
+                "function": "sha256",
+                "fingerprint": "226efd3a126fb86e71d60a37353d17f57af816d1c7ecad0623c21f0bf73eb0c7"
+            }
+        ]
+    }
```

Comment:
Invalid JSON file

Reply:
Hello,
Thank you very much on your review and suggestions, sorry that I submited in wrong place.
Could you please tell me what makes archiwarep5.json invalid (what exactly should I correct, besides release from 11.2 to 11.3)?
Additional question, if package doesn't depend on other packages can I leave packages field empty ("pkgs": [],)?
Could you tell me if there is someone I could contact if I have some additional questions regarding package creation, submission etc...
Thank you and best regards
Igor